proposals/simd/BinarySIMD.md

```diff
@@ -172,15 +172,21 @@ The `v8x16.shuffle` instruction has 16 bytes after `simdop`.
 | `v16x8.load_splat`         |    `0xc3`| -                  |
 | `v32x4.load_splat`         |    `0xc4`| -                  |
 | `v64x2.load_splat`         |    `0xc5`| -                  |
-| `i8x16.narrow_i16x8_s`     |    `0xc6`| -                  |
-| `i8x16.narrow_i16x8_u`     |    `0xc7`| -                  |
-| `i16x8.narrow_i32x4_s`     |    `0xc8`| -                  |
-| `i16x8.narrow_i32x4_u`     |    `0xc9`| -                  |
-| `i16x8.widen_low_i8x16_s`  |    `0xca`| -                  |
-| `i16x8.widen_high_i8x16_s` |    `0xcb`| -                  |
-| `i16x8.widen_low_i8x16_u`  |    `0xcc`| -                  |
-| `i16x8.widen_high_i8x16_u` |    `0xcd`| -                  |
-| `i32x4.widen_low_i16x8_s`  |    `0xce`| -                  |
-| `i32x4.widen_high_i16x8_s` |    `0xcf`| -                  |
-| `i32x4.widen_low_i16x8_u`  |    `0xd0`| -                  |
-| `i32x4.widen_high_i16x8_u` |    `0xd1`| -                  |
+| `i16x8.load8x8_u`          |    `0xc6`| m:memarg           |
+| `i16x8.load8x8_s`          |    `0xc7`| m:memarg           |
+| `i32x4.load16x4_u`         |    `0xc8`| m:memarg           |
+| `i32x4.load16x4_s`         |    `0xc9`| m:memarg           |
+| `i64x2.load32x2_u`         |    `0xca`| m:memarg           |
+| `i64x2.load32x2_s`         |    `0xcb`| m:memarg           |
```

Comment:
V8 has implementated these using the older opcodes, see https://cs.chromium.org/chromium/src/v8/src/wasm/wasm-opcodes.h?l=418-429&rcl=e576e7591361f0f32a9b02c8f67a63ef1963a33d
Any reason why you put the loads before narrow/widen? If they go after the narrow/widen then v8 won't need any change. Otherwise it's not hard to change v8's implementation too.

Reply:
The idea is that they would be next to load and splat instructions, but I don't feel very strongly about that (load and plats are not next to regular loads, for example). @tlively, @dtig, what do you think?

Reply:
I think we should throw them in wherever for now without thinking about aesthetics too much. I expect we will do a final renumbering later in the standardization process (maybe just before someone decides it's time to ship) once we have a better idea of what will make it into MVP SIMD and possibly what will be in the next iteration of SIMD.